src/components/StakingVaults/AllEarnOpportunities.tsx

```diff
@@ -31,13 +31,15 @@ export const AllEarnOpportunities = () => {
   const { cosmosStakingOpportunities } = useCosmosStakingBalances({
     assetId: 'cosmos:cosmoshub-4/slip44:118',
   })
-
+  const { cosmosStakingOpportunities: osmosisStakingOpportunities } = useCosmosStakingBalances({
+    assetId: 'cosmos:osmosis-1/slip44:118',
+  })
   const { cosmosStaking } = useModal()
 
   const allRows = useNormalizeOpportunities({
     vaultArray: sortedVaults,
     foxyArray: foxyRows,
-    cosmosStakingOpportunities,
+    cosmosStakingOpportunities: cosmosStakingOpportunities.concat(osmosisStakingOpportunities),
```

Comment:
Very nitpick so take it as you want - should `cosmosStakingOpportunities` just be `stakingOpportunities` since its not just cosmos.

Reply:
Memoized expression: https://github.com/shapeshift/web/pull/2067/commits/16d89501d12282c9acce1107d16db70c7ed8f144
Fixed naming: https://github.com/shapeshift/web/pull/2067/commits/cfaf1736496e9d344cb64ff6091e42bd0b2ff75d